src/externals/CMake/pFUnit_utils.cmake

```diff
@@ -215,7 +215,8 @@ function(create_pFUnit_test test_name executable_name pf_file_list fortran_sourc
   endif()
 
   # Prefix command with an mpirun command
-  set (MY_COMMAND ${PFUNIT_MPIRUN} ${MY_COMMAND})
+  separate_arguments(PFUNIT_MPIRUN_LIST UNIX_COMMAND ${PFUNIT_MPIRUN})
```

Comment:
This seems to be needed when PFUNIT_MPIRUN is made up of multiple, space-delimited strings - i.e., an mpirun command and some arguments.